databases/week2/schoolDatabase.sql

```diff
@@ -0,0 +1,32 @@
+create database schoolDatabase;
+use schoolDatabase;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+SET NAMES utf8mb4;
+
+CREATE TABLE `class` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `begins` DATE NOT NULL,
+   `ends` DATE NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE `student` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `email` varchar(255)  NOT NULL,
+   `phone` varchar(30) NOT NULL,
+    `class_id` int(10) unsigned, 
```

Comment:
The attribute `class_id `is not belong to the student class.
You need a `class_students `table to save the relations. A class has many students and a student has many classes.